UnityProject/Assets/Mirror/Runtime/NetworkIdentity.cs

```diff
@@ -320,7 +320,10 @@ internal void Awake()
 
             if (hasSpawned)
             {
-                Debug.LogError($"{name} has already spawned. Don't call Instantiate for NetworkIdentities that were in the scene since the beginning (aka scene objects).  Otherwise the client won't know which object to use for a SpawnSceneObject message.");
+				// Unitystation edit: add more information
+                Debug.LogError($"{name} in {transform.parent.name} at position {transform.position} has already spawned. " +
```

Comment:
```suggestion
                Debug.LogError($"{name} in {transform.parent.OrNull()?.name} at position {transform.position} has already spawned. " +
```

Could be null if theres something wrong.